composer.json

```diff
@@ -0,0 +1,14 @@
+{
+    "require-dev": {
+        "phpunit/phpunit": "^9.6"
+    },
+    "require": {
+        "silverstripe/supported-modules": "dev-pulls/main/combine-all-versions"
+    },
+    "repositories": {
+        "silverstripe/supported-modules": {
+            "type": "vcs",
+            "url": "git@github.com:creative-commoners/supported-modules.git"
+        }
+    }
```

Comment:
```suggestion
    "require": {
        "silverstripe/supported-modules": "dev-main"
    },
    "require-dev": {
        "phpunit/phpunit": "^9.6"
    }
```

Reply:
Yup, that'll be done once the PR for that fork is merged, so we can easily rerun CI here against that PR to see it go green if changes are needed over there.

Reply:
Done now that the fork PR is merged and the repo is in packagist.